shared_model/validators/signable_validator.hpp

```diff
@@ -32,13 +32,15 @@ namespace shared_model {
         auto answer = ModelValidator::validate(model);
         std::string reason_name = "Signature";
         ReasonsGroupType reason(reason_name, GroupedReasons());
-        FieldValidator().validateSignatures(
-            reason, model->signatures(), model->payload());
+        val_.validateSignatures(reason, model->signatures(), model->payload());
```

Comment:
out of that pr

Reply:
Why?

Reply:
because it's not realated to validators iface at all